ptvsd/wrapper.py

```diff
@@ -1864,6 +1866,51 @@ def version_str(v):
         }
         self.send_response(request, **sys_info)
 
+    # VS specific custom message handlers
+    @async_handler
+    def on_setDebuggerProperty(self, request, args):
+        jmc = int(args.get('JustMyCodeStepping', 0))
+        self.debug_options['JUST_MY_CODE'] = jmc > 0
+        self._apply_code_stepping_settings()
+        self.send_response(request)
+
+    def _apply_code_stepping_settings(self):
+        if self.debug_options.get('JUST_MY_CODE', False):
+            project_dirs = []
+            for path in sys.path + [os.getcwd()]:
+                is_stdlib = False
+                norm_path = os.path.normcase(path)
+                for prefix in STDLIB_PATH_PREFIXES:
+                    if norm_path.startswith(prefix):
+                        is_stdlib = True
+                        break
+                if path.endswith('ptvsd') or path.endswith('site-packages'):
```

Comment:
How do these get hit without triggered the `STDLIB_PATH_PREFIXES` check? (And why not just put our `ptvsd` path in `STDLIB_PATH_PREFIXES` if it's because it may be bundled?)

Reply:
The site-packages in the roaming directory is hard to detect, hence using explicit site-packages check (`C:\Users\kanadig\AppData\Roaming\Python\Python36\site-packages`). I had multiple, `ptvsd` folders, including one set via PYTHONPATH. This was one of the ways to get rid of all of them.

Reply:
`site.getusersitepackages()` and `site.getsitepackages()` are your friends here.

Reply:
Ah! it is get`user`sitepackages. I missed that one. I will update this.